packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -21,6 +21,21 @@ export default class RatingBar extends PureComponent {
     maxScore: 5,
   }
 
+  get ratingLegend(){
```

Comment:
Be sure to run `yarn lint` before you commit, also see if you can set up your editor to display eslint errors so you can catch them sooner. There are numerous lint violations (missing or extra spacing, etc)

Reply:
@barelyfitz will do